index.bs

```diff
@@ -782,28 +801,38 @@ spec:dom-ls; type:interface; text:Document
     dangerous and should not be used if you have Real Deal auth and csrf cookies
     used.
 
-   * '<dfn>`unsafe-include-credentials`</dfn>' All cross-origin requests to the
-     <a>physical origin</a> for the execution context will include
-     <a>credentials</a>. In particular, when this option is set in an execution
-     context, all cross-origin <a>requests</a> from that execution context to
-     the <a>physical origin</a> will have their <a>credentials mode</a> set to
-     "`include`". Additionally, if the <a>credentials mode</a> is
-     programmitically set, it will still be overriden to be set to "`include`".
-
-     Note: This applies to <em>all</em> cross-origin requests, not just
-     `xmlHttpRequests` and `fetch` requests. Thus all cross-origin elements on
-     the page, including, for example, image tags, will have the equivalement of
-     the `crossorigin` attribute set to "`use-credentials`". See <a
-     href="https://fetch.spec.whatwg.org/#cors-protocol-and-credentials">CORS
-     protocol and credentials</a> for more on including credentials and the
-     implications of including credentials in requests.
+   * '<dfn>`unsafe-credentials`</dfn>' All cross-origin requests  to the <a>physical origin</a> for the execution
+     context will include <a>credentials</a>.
+
+     Specifically, update the step 2 of <a>HTTP-network-or-cache fetch</a> in
+     the Fetch spec [[!FETCH]] to read:
+
+     2. Let credentials flag be set if one of
+        * `request`'s credentials mode is "include"
+        * `request`'s credentials mode is "same-origin" and request's response
+          tainting is "basic"
+        * `request`'s credentials mode is "same-origin" and `request`'s
+          <a>environment settings object</a> has the `suborigin unsafe
+          credentials` flag set and the request's <a>current url</a> is
+          <a>same-physical-origin</a> with <a>request origin</a>.
+
+     Note: This has several important, practical effects. If the <a>credentials
```

Comment:
I wonder if worth clarifying that "and creds will be set even for requests to another sub-origin in the same physical origin"